build.gradle

```diff
@@ -114,12 +114,13 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-cache'
 
     implementation 'org.springframework.boot:spring-boot-starter-security'
+
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-resource-server'
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
     implementation 'org.springframework.security:spring-security-oauth2-jose'
+
     implementation 'org.springframework:spring-webflux'
     implementation 'org.springframework.boot:spring-boot-starter-reactor-netty'
-    implementation 'com.nimbusds:nimbus-jose-jwt:8.10' // pinned due to CVE-2019-17195 in 7.8.1 brought in by spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE
```

Comment:
Does this not need to be pinned any more? I thought 2.2.6 still pulls in an affected version of this?

Reply:
Just double checked and 2.2.6 pulls in com.nimbusds:nimbus-jose-jwt:8.8 so think we're fine.

Reply:
Ah cool, ok! Perhaps I was thinking of something else.